2022-05-11 12:57:23,415 INFO  ==============================================================
2022-05-11 12:57:23,416 INFO  Id:    SoftwareManagement.sm_docker_install_remove_multiple
2022-05-11 12:57:23,417 INFO  Dir:   software_management_end_to_end/
2022-05-11 12:57:23,418 INFO  Title: Validate end to end behaviour for the docker plugin for multiple docker images with mixed versions
2022-05-11 12:57:23,419 INFO  ==============================================================
2022-05-11 12:57:23,430 INFO  Using sw id database: {'asciijump': '5475369', 'robotfindskitten': '5474869', 'squirrel3': '5475279', 'rolldice': '5152439', 'moon-buggy': '5439204', 'apple': '5495053', 'banana': '5494888', 'cherry': '5495382', 'watermelon': '5494510', 'registry': '8018911', 'hello-world': '8021526', 'docker/getting-started': '8021973', 'alpine': '7991792'}
2022-05-11 12:57:23,431 INFO  EnvironmentC8y Setup
2022-05-11 12:57:23,993 INFO  Executed systemctl<serv_mapper1>, exit status 3
2022-05-11 12:57:24,100 INFO  Restarting mosquitto too frequently in the last 10 seconds. It was only up for 5 seconds
2022-05-11 12:57:24,101 INFO  Delaying execution by 6 seconds
2022-05-11 12:57:44,114 INFO  Executed sudo<tedge_connect_c8y>, exit status 0, duration 14 secs
2022-05-11 12:57:44,732 INFO  Executed sudo<tedge_connect_c8y_test>, exit status 0
2022-05-11 12:57:45,297 INFO  Executed systemctl<serv_mosq2>, exit status 0
2022-05-11 12:57:45,760 INFO  Executed systemctl<serv_mapper3>, exit status 0
2022-05-11 12:57:48,000 INFO  State of current operation: SUCCESSFUL
2022-05-11 12:57:48,221 INFO  Response status: 201
2022-05-11 12:57:48,223 INFO  Response to action: {
    "creationTime": "2022-05-11T12:57:48.177Z",
    "deviceName": "m32sd10c",
    "deviceId": "18764682",
    "self": "https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/18857965",
    "id": "18857965",
    "status": "PENDING",
    "description": "Apply software changes, triggered from PySys: [{'action': 'install', 'id': '8021526', 'name': 'hello-world', 'url': '', 'version': '::docker'}, {'action': 'install', 'id': '8018911', 'name': 'registry', 'url': '', 'version': '2.6.2::docker'}]",
    "c8y_SoftwareUpdate": [
        {
            "name": "hello-world",
            "action": "install",
            "id": "8021526",
            "version": "::docker",
            "url": ""
        },
        {
            "name": "registry",
            "action": "install",
            "id": "8018911",
            "version": "2.6.2::docker",
            "url": ""
        }
    ]
}
2022-05-11 12:57:48,224 INFO  Started operation: {'creationTime': '2022-05-11T12:57:48.177Z', 'deviceName': 'm32sd10c', 'deviceId': '18764682', 'self': 'https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/18857965', 'id': '18857965', 'status': 'PENDING', 'description': "Apply software changes, triggered from PySys: [{'action': 'install', 'id': '8021526', 'name': 'hello-world', 'url': '', 'version': '::docker'}, {'action': 'install', 'id': '8018911', 'name': 'registry', 'url': '', 'version': '2.6.2::docker'}]", 'c8y_SoftwareUpdate': [{'name': 'hello-world', 'action': 'install', 'id': '8021526', 'version': '::docker', 'url': ''}, {'name': 'registry', 'action': 'install', 'id': '8018911', 'version': '2.6.2::docker', 'url': ''}]}
2022-05-11 12:57:50,483 INFO  State of operation 18857965 : PENDING
2022-05-11 12:57:52,688 INFO  State of operation 18857965 : EXECUTING
2022-05-11 12:57:54,907 INFO  State of operation 18857965 : EXECUTING
2022-05-11 12:57:57,121 INFO  State of operation 18857965 : EXECUTING
2022-05-11 12:57:59,332 INFO  State of operation 18857965 : FAILED
2022-05-11 12:57:59,333 ERROR Stopping as the operation has failed
2022-05-11 12:57:59,334 WARN  SystemError occurred while running test - The operation has failed
Traceback (most recent call last):
  File "/home/pi/env-pysys/lib/python3.7/site-packages/pysys/baserunner.py", line 1292, in __call__
    self.testObj.setup()
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/sm_docker_install_remove_multiple/run.py", line 55, in setup
    self.wait_until_succcess()
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/environment_sm_management.py", line 324, in wait_until_succcess
    self.wait_until_status("SUCCESSFUL")
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/environment_sm_management.py", line 361, in wait_until_status
    raise SystemError("The operation has failed")
SystemError: The operation has failed
2022-05-11 12:57:59,358 WARN  SystemError: The operation has failed ... blocked 
2022-05-11 12:57:59,359 INFO  
2022-05-11 12:57:59,360 INFO  cleanup:
2022-05-11 12:58:03,162 INFO  Executed sudo<tedge_disconnect_c8y>, exit status 0
2022-05-11 12:58:03,725 INFO  Executed systemctl<serv_mapper5>, exit status 3
2022-05-11 12:58:03,730 INFO  
2022-05-11 12:58:03,731 INFO  Test duration: 40.33 secs
2022-05-11 12:58:03,732 INFO  Test final outcome:  BLOCKED
2022-05-11 12:58:03,733 INFO  Test outcome reason: SystemError: The operation has failed
2022-05-11 12:58:03,733 INFO  
